nullaway/src/test/java/com/uber/nullaway/NullAwayFrameworkTests.java

```diff
@@ -365,4 +365,75 @@ public void systemConsoleNullable() {
             "}")
         .doTest();
   }
+
+  @Test
+  public void mapGetOrDefault() {
+    makeTestHelperWithArgs(
+            Arrays.asList(
+                "-d",
+                temporaryFolder.getRoot().getAbsolutePath(),
+                "-XepOpt:NullAway:AnnotatedPackages=com.uber"))
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import java.util.Map;",
+            "import com.google.common.collect.ImmutableMap;",
+            "import org.jspecify.annotations.Nullable;",
+            "class Test {",
+            "   void testGetOrDefaultMap(Map<String, String> m, String string, @Nullable String nullableString) {",
```

Comment:
Minor nit, but could we call the first string `nonNullString` or similar? If not, let's call it `s`. I know it's perfectly legal, but lower-case "string" just seems like a typo for a fraction of a second every time I read over this line:

```
m.getOrDefault(\"key\", string).toString();
```

Reply:
Fixed in c27f3e3bee0040be87222305c252e121f9f2228c